generators/app/index.js

```diff
@@ -43,6 +43,15 @@ module.exports = class extends Generator {
         message: 'What\'s the name of your project?',
         default: 'Toolbox'
       }, {
+        type: 'input',
+        name: 'remote',
+        message: 'What\'s the URL of your repo?',
+        default: function (answers) {
```

Comment:
You ask for the full URL and you concat it with Antistatique organisation repo url ? 🤔  Not sure to understand this one....